library/sn3218.py

```diff
@@ -17,6 +9,26 @@
 CMD_UPDATE = 0x16
 CMD_RESET = 0x17
 
+if sys.version_info < (3, ):
+    SMBUS = "python-smbus"
+else:
+    SMBUS = "python3-smbus"
+
+# Helper function to avoid exception chaining output in python3.
+# use exec to shield newer syntax from older python
+if sys.version_info < (3, 3):
```

Comment:
raise from None only appears in py3.3. Before that it's not worthwhile, so old py3 versions might see the exception chain.